iaso/tests/api/test_org_units_bulk_update.py

```diff
@@ -246,6 +246,43 @@ def test_org_unit_bulkupdate_select_all(self):
 
         self.assertEqual(5, am.Modification.objects.count())
 
+    @tag("iaso_only")
+    def test_org_unit_bulkupdate_select_all_with_restricted_write_permission_for_user(self):
+        """
+        Check that we cannot bulk edit all org units if writing rights are limited
+        by a set of org unit types that we are allowed to modify.
+        """
+        self.yoda.iaso_profile.editable_org_unit_types.set(
+            # Only org units of this type are now writable.
+            [self.jedi_squad]
+        )
+        self.client.force_authenticate(self.yoda)
+
+        response = self.client.post(
+            f"/api/tasks/create/orgunitsbulkupdate/",
+            data={"select_all": True, "validation_status": m.OrgUnit.VALIDATION_VALID},
+            format="json",
+        )
+        self.assertJSONResponse(response, 201)
+        data = response.json()
+        task = self.assertValidTaskAndInDB(data["task"], status="QUEUED", name="org_unit_bulk_update")
+        self.assertEqual(task.launcher, self.yoda)
+
+        # Run the task.
+        self.runAndValidateTask(task, "SUCCESS")
+
+        for org_unit in [self.jedi_squad_endor_1, self.jedi_squad_endor_2]:
+            org_unit.refresh_from_db()
+            self.assertEqual(org_unit.validation_status, m.OrgUnit.VALIDATION_VALID)
```

Comment:
They already were in `VALIDATION_VALID` so it's hard to determine if the bulk update happened. I'd make the bulk update to `VALIDATION_NEW` or `VALIDATION_REJECTED`

Reply:
Fixed in https://github.com/BLSQ/iaso/pull/1691/commits/c42d1cc9237799dbb581cc937bc18eb54c1c8f68